draft-ietf-core-coral.md

```diff
@@ -165,6 +167,60 @@ In contrast to RDF, however, the focus of CoRAL is not on the
 description of a graph of resources, but on the discovery of possible
 future application states.
 
+-->
+### Definitions
+
+The *CoRAL information model* is similar to the [Resource Description Framework (RDF)](#W3C.REC-rdf11-concepts-20140225) information model:
+Data is expressed as an (unordered) set of triples (also called statements),
+consisting of a subject, a predicate and an object.
+The predicate is always a URI in syntax normalized form, and subject and object are either a URI in syntax normalized form, or a concrete CBOR object (a literal).
+
+These triples form a directed graph with the subject and object being source and destination, and the predicate a description on the edge.
+That graph is equivalent to the data.
+
+To form a set and a graph, we define an equivalence relation:
```

Comment:
not sure what "set" are we talking about here?
is it the data set?  but if so, why do we need to form both a graph *and* a set: isn't the graph a synonym of dataset (as per L179)?